deployments/brev/README-k8s.md

```
@@ -0,0 +1,167 @@
+2. Navigate to your OSMO instance
+3. Select "Access"
+4. Under `Using Secure Links`, click `Share a Service` and enter port `30080`
+5. Click on the "Shareable URL" for port `30080`
+
+## [Optional] Local CLI Setup
+
+To use the OSMO CLI and UI from your local machine, set up port forwarding and install the necessary tools.
+
+### Step 1: Install Brev CLI
+
+Follow instructions [here](https://docs.nvidia.com/brev/latest/brev-cli.html#installation-instructions). Be sure to `brev login`.
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)